vtex/pickup-points-modal
# Update markers to change size when hovers list and a pickup point is selected

#### What is the purpose of this pull request?
### Fixed

- Marker changes size on hover and when a pickup point in the list is selected

#### How should this be manually tested?
1. Add [items to cart](https://fernando--drimjuguetes.myvtex.com/checkout/cart/add?&workspace=fernando&sku=63525&qty=1&seller=1&sc=1)
2. Go to shipping and select pickup tab
3. Search for `Barcelona`
4. Hover in the pickups list and related markers should change size

#### Screenshots or example usage
n/a
#### Types of changes
